scripts/ingest.rb

```diff
@@ -1,8 +1,10 @@
 require_relative '../lib/rails_stub'
 require_relative 'lib/pb_core_ingester'
-require 'logger'
+require_relative '../lib/has_logger'
```

Comment:
unless it's needed by the runtime rails code, could this just be 'lib/has_logger'? Compare with the line above.